doc/source/whatsnew/v2.0.0.rst

```diff
@@ -185,6 +185,7 @@ Performance improvements
 - Performance improvement in :func:`merge` and :meth:`DataFrame.join` when joining on a sorted :class:`MultiIndex` (:issue:`48504`)
 - Performance improvement in :meth:`DataFrame.loc` and :meth:`Series.loc` for tuple-based indexing of a :class:`MultiIndex` (:issue:`48384`)
 - Performance improvement for :meth:`MultiIndex.unique` (:issue:`48335`)
+- Minimize memory usage when writing BZ2 & LZMA files by using out-of-band pickling (:issue:`49068`)
```

Comment:
Would be good to reference DataFrame/Series.to_pickle in the whatsnew

Reply:
```suggestion
- Reduce memory usage of DataFrame/Series `to_pickle` when using BZ2 or LZMA (:issue:`49068`)
```